dashboard/test_dashboard.py

```diff
@@ -0,0 +1,7 @@
+# pylint: skip-file
+
+from os import path
```

Comment:
Smart idea with the placeholder test. This can be removed when actual code is written.